admiral/pkg/clusters/configwriter.go

```diff
@@ -112,36 +112,61 @@ func getIngressEndpoints(clusters map[string]*registry.IdentityConfigCluster) (m
 
 // getServiceEntryEndpoint constructs the remote or local endpoints of the service entry that
 // should be built for the given identityConfigEnvironment.
-func getServiceEntryEndpoint(
+func getServiceEntryEndpoints(
 	ctxLogger *logrus.Entry,
 	clientCluster string,
 	serverCluster string,
 	ingressEndpoints map[string]*networkingV1Alpha3.WorkloadEntry,
-	identityConfigEnvironment *registry.IdentityConfigEnvironment) (*networkingV1Alpha3.WorkloadEntry, error) {
-	//TODO: Verify Local and Remote Endpoints are constructed correctly
+	identityConfigEnvironment *registry.IdentityConfigEnvironment) ([]*networkingV1Alpha3.WorkloadEntry, error) {
+	if len(identityConfigEnvironment.Services) == 0 {
+		return nil, fmt.Errorf("there were no services for the asset in namespace %s on cluster %s", identityConfigEnvironment.Namespace, serverCluster)
```

Comment:
This is more common, maybe make this info

Reply:
Noted, will put this in next PR